include/engine/trip/trip_brute_force.hpp

```diff
@@ -25,62 +26,77 @@ namespace trip
 EdgeWeight ReturnDistance(const util::DistTableWrapper<EdgeWeight> &dist_table,
                           const std::vector<NodeID> &location_order,
                           const EdgeWeight min_route_dist,
-                          const std::size_t component_size)
+                          const std::size_t number_of_locations)
 {
     EdgeWeight route_dist = 0;
     std::size_t i = 0;
     while (i < location_order.size() && (route_dist < min_route_dist))
     {
-        route_dist += dist_table(location_order[i], location_order[(i + 1) % component_size]);
-        BOOST_ASSERT_MSG(dist_table(location_order[i], location_order[(i + 1) % component_size]) !=
-                             INVALID_EDGE_WEIGHT,
-                         "invalid route found");
+
```

Comment:
A better naming for `i` would could be `current_index` and for `next_index = (current_index + 1) % number_of_locations`.